test/integration_test.go

```diff
@@ -107,8 +108,9 @@ func woodpeckerRun(conf woodpecker.Config, fetchIterations int) (string, string,
 	woodpecker.Run()
 
 	// Sleep for the right amount of time based on the fetchInterval and the
-	// requested number of iterations
-	time.Sleep(fetchInterval * time.Duration(fetchIterations))
+	// requested number of iterations.
+	jitter := time.Millisecond * 20
```

Comment:
Nit: This isn't really jitter. Could we call it buffer or padding instead?

Reply:
Sure, makes sense to me. Renamed in 25e1562 and I've updated the PR title. Thanks @jsha